utils/DeployUtils.ps1

```diff
@@ -244,12 +272,12 @@ function ConfigureScubaGearModule {
         $ErrorActionPreference = $CurrentErrorActionPreference
     }
     catch {
-        Write-Warning "Error: Cannot update module manifest:"
-        Write-Warning "Stacktrace:"
-        Write-Warning $_.ScriptStackTrace
-        Write-Warning "Exception:"
+        Write-Warning ">>> Error: Cannot update module manifest:"
+        # Write-Warning ">>> Stacktrace:"
+        # Write-Warning $_.ScriptStackTrace
+        Write-Warning ">>> Exception:"
         Write-Warning $_.Exception
-        Write-Error "Failed to update module manifest"
+        # Write-Error ">>> Failed to update the module manifest."
```

Comment:
```suggestion
```

Reply:
That was actually a typo.  Good catch.  The Write-Error was correct code.